recipes/nodejs_from_source.rb

```diff
@@ -24,9 +24,15 @@
 
 case node['platform_family']
 when 'rhel', 'fedora', 'amazon'
-  package %w(openssl-devel python3 tar)
+  directory "node['nodejs']['tmpdir']"
+  package %w(openssl-devel python2 tar)
```

Comment:
I think it might be good to add a comment above this explaining that the installer only works with python2 currently.

Reply:
Added